lighthouse-core/test/results/sample_v2.json

```diff
@@ -3473,14 +3496,54 @@
       "scoreDisplayMode": "numeric",
       "numericValue": 0,
       "numericUnit": "millisecond",
-      "displayValue": "",
+      "displayValue": "Potential savings of 49 KiB",
```

Comment:
it's kind of hard to tease apart which changes are from the the new artifacts or if we accidentally broke something. did you look into these new images? did they not exist in the previous artifacts?

Reply:
Talked about this offline a bit, the PR doesn't seem to affect 👍